pkg/packagekit/updates.jsx

```diff
@@ -235,6 +235,25 @@ function HeaderBar(props) {
     );
 }
 
+function getSeverityURL(urls) {
+    // in ascending severity
+    const knownLevels = ["low", "moderate", "important", "critical"];
+    var highestIndex = -1;
+    var highestURL = null;
+
+    // search URLs for highest valid severity; by all means we expect an update to have at least one, but for paranoia..
```

Comment:
"exactly one"?

Reply:
On RHEL we should usually have an erratum and a severity attached, but this is not the case on other distributions. So "at most one" is correct.

Reply:
and right, it says "at least", it should be "at most" :-) . Fix pushed.

Reply:
Right, thanks!